### PR TITLE
Add padding for Subscription Status & upload

### PR DESCRIPTION
**Pull request by alex-deaconu**
[stage-4]

### CODE REVIEW

File: web/partials/storage/storage-modal.html [hunk 1]
```diff
@@ -1,66 +1,65 @@
...
+    	<div class="app-header">
+        <div ng-controller="SubscriptionStatusController" class="add-left add-right">
+          <div id="subscription-status" subscription-status
+               product-id="{{productId}}" product-code="{{productCode}}" company-id="{{companyId}}"
+               ng-model="subscriptionStatus.result" show-store-modal="showStoreModal" expanded-format="true">
+          </div>
+        </div>
+        
+        <div class="title-block"></div>
+        <ng-include src="'partials/storage/global-actions.html'"></ng-include>
```
Comment (alex-deaconu) [inline web/partials/storage/storage-modal.html]: Sorry about the spacing changes. 

Basically I moved the subscription-status & global-actions into the `app-header` div. Also added `add-left add-right` to the subscription status.

Thanks.

**Comment by alex-deaconu** [comment]
@ghislaineguerin @RV-Adrian I've added the classes as discussed. Similar to the subscription-status the upload status panel was also slightly wider than the rest.

Updated for both full Storage and the Modal.

@ezequielc please review. Thanks!

**Comment by RV-Adrian** [comment]
@alex-deaconu thx!

**Comment by ezequielc** [comment]
@alex-deaconu :+1: Thanks for fixing!

**Pull request merged**